jekyll/_cci2/config-policy-management-overview.adoc

```diff
@@ -605,7 +347,7 @@ circleci policy push ./policy_bundle_dir_path --owner-id 462d67f8-b232-4da4-a7de
 
 Config Policy Management is a beta feature. If this feature interests you please contact us to participate in the beta. 
 
-# TODO: figure out a clearer path for "contact us" - maybe Idoh would know
+#TODO: figure out a clearer path for "contact us" - maybe Idoh would know#
```

Comment:
@sagar-connect could you check if this "To do" has been addressed?

Reply:
@gersteni has suggested that we use his circleci.com email address (not posting here since this is a public repo)

Reply:
Docs is all open source so if we add it it will be public 🤷